src/github/handlers/issues/closed.ts

```diff
@@ -0,0 +1,12 @@
+import { GitHubContext } from "../../github-context";
+
+export async function issuesClosed(event: GitHubContext<"issues.closed">) {
+  if (event.payload.issue.state_reason === "not_planned") {
```

Comment:
why is this in the kernel? it should be handled in the plugin

Reply:
Right it doesn't make sense to have it here. However I do recall that @0x4007 wanted the comments to be generated by the kernel and not the plugin, so I believe we will need communication back and forth.